eventkit_cloud/tasks/task_factory.py

```diff
@@ -64,125 +61,76 @@ def parse_tasks(self, worker=None, run_uid=None, user_details=None):
-            grouped_provider_tasks.append([osm_generic_provider_task, osm_thematic_provider_task])
-
-            if grouped_provider_tasks:
-                tasks_results = []
-                # Contains one chain per item in grouped_provider_tasks
-                grouped_provider_subtask_chains = []
-                for grouped_provider_task in grouped_provider_tasks:
-                    export_provider_task_uids = []
-                    provider_subtask_chains = []
-                    for provider_task in grouped_provider_task:
-                        if not provider_task:
-                            continue
-                        # Create an instance of a task runner based on the type name
-                        if self.type_task_map.get(provider_task.provider.export_provider_type.type_name):
-                            type_name = provider_task.provider.export_provider_type.type_name
-                            task_runner = self.type_task_map.get(type_name)()
-                            os.makedirs(os.path.join(run_dir, provider_task.provider.slug), 6600)
-                            stage_dir = os.path.join(
-                                        run_dir,
-                                        provider_task.provider.slug)
-
-                            args = {
-                                'user': job.user,
-                                'provider_task_uid': provider_task.uid,
-                                'run': run,
-                                'stage_dir': stage_dir,
-                                'service_type': provider_task.provider.export_provider_type.type_name,
-                                'worker': worker,
-                                'user_details': user_details
-                            }
-
-                            export_provider_task_uid, chain_task_runner_tasks = task_runner.run_task(**args)
-                            export_provider_task_uids += [export_provider_task_uid]
-
-                            if chain_task_runner_tasks:
-                                # Create a geojson for clipping this provider if needed
-                                selection_task = create_task(
-                                    export_provider_task_uid=export_provider_task_uid, stage_dir=stage_dir,
-                                    worker=worker, task=output_selection_geojson_task, selection=job.the_geom.geojson,
-                                    user_details=user_details
-                                )
-
-                                # Export the bounds for this provider
-                                bounds_task = create_task(
-                                    export_provider_task_uid=export_provider_task_uid, stage_dir=stage_dir,
-                                    worker=worker, task=bounds_export_task, user_details=user_details
-                                )
-
-                                if provider_task.provider.zip:
```

Comment:
This is the part that helps make the QGIS stuff worked.  If a provider is zipped it bundles up the QGIS, bounds, and Style icons in a zip file. We should make sure that is working before we switch over to this.